app/controllers/provider_interface/condition_statuses_controller.rb

```diff
@@ -74,5 +68,13 @@ def success_message
         'Status of conditions updated'
       end
     end
+
+    def wizard_controller_excluded_paths
+      []
+    end
+
+    def wizard_flow_controllers
+      ['provider_interface/condition-statuses', 'provider_interface/condition_statuses/checks'].freeze
```

Comment:
Should this be?
```suggestion
      ['provider_interface/condition_statuses', 'provider_interface/condition_statuses/checks'].freeze
```

Reply:
Good catch. It was causing the wizard to clear on refresh which is now fixed.